lib/README

```diff
@@ -160,3 +160,7 @@ revision 177b0073fe6f19281ee7f7fdbe9599e32d1b4b8b.
 The Huada HC32F460 directory contains code from:
   https://www.hdsc.com.cn/Category83-1490
 version 2.2 DDL minus example directory, empty/extra files
+
+The n32g45x directory contains parts of code from:
+  https://github.com/RT-Thread/rt-thread/tree/master/bsp/n32g452xx/Libraries/N32_Std_Driver
+version v1.0.1
```

Comment:
For sources from github, please include the full revision (as seen in other github references in this README file).